doc/source/index.rst

```diff
@@ -368,6 +389,10 @@ Example
       - avhrr-3
 
     min_coverage: 25
+    s3_config:
+      target: s3://name-of-the-bucket/
+      delete_files: true
```

Comment:
Do we still need `output_dir` when we use `s3_config` like this?  It would just be a temporary directory like `staging_zone`?  It would seem that this could be optional if `delete_files` is `True`, and that if the user has not defined what temporary directory to use, trollflow2 could create its own with `tempfile.TemporaryDirectory`?

Reply:
The `output_dir` is used by the `save_datasets` plugin, and defaults to current directory. The `s3_uploader` just gets the filenames from the product list and uploads them to the target bucket. Making the `s3_config` to affect the `save_datasets` would make the usage quite complex. At least complex to explain.